### PR TITLE
add supervisors guide and update menu links accordingly

### PR DESCRIPTION
**Pull request by alicetragedy**
not sure if we need this, so please review :) 
we can always turn it into a FAQ for supervisors if we want to go with an actual selection rather than accepting submissions from possible supervisors.

### CODE REVIEW

File: guide/supervisors.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+current: supervisors
+permalink: /guide/supervisors/
+---
+
+<h1>Becoming a supervisor</h1>
+
+<p>So you'd like to get involved, but you don't have any experience in coaching? Maybe you'd like to be a supervisor instead? Read below to see if it fits your profile. 
+
+<ul>
+<li><a href="what-it-takes">What it takes to be a good supervisor</a></li>
```
Comment (carpodaster) [inline guide/supervisors.html]: I think these internal links need a '#'

**Comment by alicetragedy** [comment]
@carpodaster hmmmmyes, I think you are definitely right. Oops! thank you for noticing, will change those right away.
@anikalindtner yes, we should decide on one or the other. in the guide folder the only .md file was the coaching companies one, so went along with the majority, but I think we can change all of them to markdown though!

**Comment by alicetragedy** [comment]
if there are no more change to be made to this guide specifically, we could merge (and update all the files to markdown at once outside of this PR) wdyt?

**Comment by carpodaster** [comment]
+1 for renaming the file extensions in another PR/commit (there should be no harm done: HTML is valid markdown, just not the other way round)

Thanks for the work on the supervisor guide! Merge it!!!1111 :smile:

**Comment by alicetragedy** [comment]
:tada:

**Pull request merged**